inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -338,7 +344,7 @@ CREATE TABLE IF NOT EXISTS `inlong_stream_field`
     `rank_num`            smallint(6)  DEFAULT '0' COMMENT 'Field order (front-end display field order)',
     `is_deleted`          int(11)      DEFAULT '0' COMMENT 'Whether to delete, 0: not deleted, > 0: deleted',
     PRIMARY KEY (`id`),
-    KEY `field_stream_id_index` (`inlong_stream_id`)
+    KEY `group_field_index` (`inlong_group_id`, `inlong_stream_id`)
```

Comment:
Suggest changing the `KEY` to `INDEX`, because some DBMS will not add an index for KEY.